_sources/Sequences/StringsandLists.rst

```diff
@@ -0,0 +1,71 @@
+..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
+    Meyers, and Dario Mitchell.  Permission is granted to copy, distribute
+    and/or modify this document under the terms of the GNU Free Documentation
+    License, Version 1.3 or any later version published by the Free Software
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: sequences-2-
+   :start: 1
+
+Strings and Lists
+=================
+
+Throughout the first chapters of this book we have used strings to represent words or phrases that we
+wanted to print out. Our definition was simple: a string is simply some characters inside quotes.  
+In this chapter we explore strings in much more detail.
+
+Additionally, we explore lists, which are very much like strings but can hold different types. In fact, you can think of a string as a list of characters!
```

Comment:
I would delete the sentence that says you can think of a string as a list of characters.